health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Summary.js

```
@@ -0,0 +1,117 @@
+        <div
+          style={{
+            width: "100%",
+            display: "flex",
+            justifyContent: "end",
+          }}
+        >
+          {/* {data?.insight?.indicator === "upper_green" ? ArrowUpwardElement("10px") : ArrowDownwardElement("10px")} */}
+          <p className={`${data?.insight.colorCode}`} style={{ whiteSpace: "pre" }}>
+            {insight?.[0] &&
+              `${Digit.Utils.dss.formatter(insight[0], "number", value?.denomination, true, t)}% ${t(
+                Digit.Utils.locale.getTransformedLocale("DSS" + insight?.[1] || "")
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)